src/api/cvc4cpp.h

```diff
@@ -575,11 +575,26 @@ class CVC4_PUBLIC Term
    */
   bool operator!=(const Term& t) const;
 
+  /**
+   * @return the id of this term
+   */
+  uint64_t getId() const;
```

Comment:
We should add a comment that this operation is not allowed on a null term.

Reply:
We currently don't have this for any of the member functions where we guard against a null Term (or Sort, for that matter). What is your preferred way to document this?

Reply:
We decided offline that this is out of scope for this PR. We postpone this since documentation about error cases is in general missing for API classes and their functions:
https://github.com/CVC4/CVC4/issues/3361